index.d.ts

```diff
@@ -1,118 +0,0 @@
-// Type definitions for launchdarkly-node-server-sdk-redis
-
-/**
- * Interface for the Redis feature store component to be used with the LaunchDarkly SDK.
- *
- * See: https://docs.launchdarkly.com/sdk/features/database-integrations
- */
-
-declare module 'launchdarkly-node-server-sdk-redis' {
```

Comment:
This was the interface file that we were previously providing for TS developers who consumed this package. The exported interface from `index.ts` should be exactly equivalent to this.